module/VuFind/src/VuFind/RecordDriver/EDS.php

```diff
@@ -74,6 +76,91 @@ public function getUniqueID()
         return $dbid . ',' . $an;
     }
 
+    /**
+     * Return the rtac identifier of this record from EDS API;
+     * RTAC ID is basically the AN without the catalog prefix
+     *
+     * @return string unique rtac identifier
+     */
+    public function getRtacIdentifier()
+    {
+        $dbid = $this->fields['Header']['DbId'];
+        $an = $this->fields['Header']['An'];
+        $catId = $this->recordConfig?->Catalog?->CatalogDatabaseId ?? '';
+
+        $regexArray = $this->recordConfig?->Catalog?->CatalogANRegex ?? [];
+        $replaceArray = $this->recordConfig?->Catalog?->CatalogANReplace ?? [];
+
+        if ($dbid === $catId && $this->pubTypeRtacEnabled()) {
+            $returnValue = $an;
+            for ($i = 0; $i < count($regexArray); $i++) {
+                $returnValue =  preg_replace($regexArray[$i], $replaceArray[$i], $returnValue);
+            }
+            return $returnValue;
+        }
+        return $dbid . ',' . $an;
+    }
+
+    /**
+     * Identify if config tells us to expect a catalog, if catalog id is set
+     * and if catalog id matches databaseid
+     *
+     * @return boolean
```

Comment:
We typically use short-form types in comments.

```suggestion
     * @return bool
```